package.json

```diff
@@ -13,7 +13,7 @@
   "scripts": {
     "build": "tsc",
     "start": "node dist/index.js",
-    "test": "jest"
+    "test": "jest --runInBand"
```

Comment:
4 speed ?

Reply:
The route tests were racing to start a server on the same port, and I didn't bother figuring out how run everything concurrently except those two with each other.
I just realized I _could_ have route tests start the server on different ports, but I think I was already also getting spurious failures running the tests concurrently on a non-`mock` redis.